packages/backend/src/models/SpaceModel.ts

```diff
@@ -1029,11 +1029,13 @@ export class SpaceModel {
             }),
         );
 
-        const rootSpaceUuids = spacesWithRootSpaceUuid
-            .filter(
-                ({ rootSpaceUuid, spaceUuid }) => rootSpaceUuid === spaceUuid,
-            )
-            .map(({ rootSpaceUuid }) => rootSpaceUuid);
+        const rootSpaceUuids = Array.from(
+            new Set(
+                spacesWithRootSpaceUuid.map(
+                    ({ rootSpaceUuid }) => rootSpaceUuid,
+                ),
+            ),
+        );
```

Comment:
The bug here was, if you passed _only_ child spaces (`spaceUuids`), all of them will be filtered out

Reply:
which meant, non-admin users would not see any content in a nested space